libc-bottom-half/headers/private/preview2.h

```diff
@@ -0,0 +1,2481 @@
+// Generated by `wit-bindgen` 0.16.0. DO NOT EDIT!
```

Comment:
Should this file live alongside the existing `./libc-bottom-half/headers/public/wasi/api.h` for preview1?

Perhaps we can rename the existing `api.h` to `preview1.h`?

Reply:
Unfortunately, `#include <wasi/api.h>` is public interface for preview1; we'd break users if we renamed it.